docs/README.md

```diff
@@ -115,3 +115,18 @@ Overview docs; Reference docs; API and chaincode developer docs; Network operati
      - Deploy Transaction
      - Execute Transaction
      - Chaincode Execution
+
+### Release Notes
+* [Fabric Releases](releasenotes/Fabric-Releases.md)
+* [Current API Changes](releasenotes/Sprint-Release-Notes.md)
+* [Release Process](releasenotes/Release-Process.md)
+
+### Proposals
```

Comment:
@tamasblummer I also think that given that we are publishing a pretty-print version of the docs from this tree that this isn't the appropriate location for the proposals. I would recommend a separate tree eg fabric/proposals. I would also propose that we have a top-level README that describes the process. I'd be happy to take this to email with a proposed readme describing the process.